ignite/engine/engine.py

```diff
@@ -641,25 +641,51 @@ def switch_batch(engine):
         self.should_terminate = self.should_terminate_single_epoch = False
 
         try:
-            self._logger.info("Engine run starting with max_epochs={}.".format(max_epochs))
+            self.logger.info("Engine run starting with max_epochs={}.".format(max_epochs))
             start_time = time.time()
             self._fire_event(Events.STARTED)
             while self.state.epoch < max_epochs and not self.should_terminate:
                 self.state.epoch += 1
                 self._fire_event(Events.EPOCH_STARTED)
                 hours, mins, secs = self._run_once_on_dataset()
-                self._logger.info("Epoch[%s] Complete. Time taken: %02d:%02d:%02d", self.state.epoch, hours, mins, secs)
+                self.logger.info("Epoch[%s] Complete. Time taken: %02d:%02d:%02d", self.state.epoch, hours, mins, secs)
                 if self.should_terminate:
                     break
                 self._fire_event(Events.EPOCH_COMPLETED)
 
             self._fire_event(Events.COMPLETED)
             time_taken = time.time() - start_time
             hours, mins, secs = _to_hours_mins_secs(time_taken)
-            self._logger.info("Engine run complete. Time taken %02d:%02d:%02d" % (hours, mins, secs))
+            self.logger.info("Engine run complete. Time taken %02d:%02d:%02d" % (hours, mins, secs))
 
         except BaseException as e:
-            self._logger.error("Engine run is terminating due to exception: %s.", str(e))
+            self.logger.error("Engine run is terminating due to exception: %s.", str(e))
             self._handle_exception(e)
 
         return self.state
+
+    def change_logger_name(self, desired_name: str):
```

Comment:
I would prefer previous setup method than 3 rather trivial methods. User can change logger name itself with
```python
engine.logger.name = "new_name"
```
without using `change_logger_name`.
Same for logging level and adding handlers...